exercicios/para-casa/entregas/AmandaLuiza/resolucao.js

```diff
@@ -0,0 +1,57 @@
+// Classe base Personagem
+class Personagem {
```

Comment:
Uma sugestão é adicionar as propriedades privadas e os métodos get e set. Eu fiz isso no meu código dessa maneira:

```suggestion
class Personagem {
  #nome;
  #nomeVidaReal;
  #poderes = [];

  constructor(nome, nomeVidaReal) {
    this.#nome = nome;
    this.#nomeVidaReal = nomeVidaReal;
  }

  get nome() {
    return this.#nome;
  }

  get nomeVidaReal() {
    return this.#nomeVidaReal;
  }
  
  ...
}

```

Reply:
Blz, vlw mesmo viu